WindowsFirewallHelper/WindowsFirewallHelper.csproj

```diff
@@ -22,7 +22,6 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <Reference Condition="'$(TargetFramework)'=='net4'" Include="System.ServiceProcess"></Reference>
```

Comment:
move this next to the new conditions too. being separated like this makes it hard to know what is exactly referenced and in what environment.